facturx/flavors/factur-x/xml/basic.xml

```diff
@@ -1,165 +1,165 @@
 <rsm:CrossIndustryInvoice xmlns:qdt="urn:un:unece:uncefact:data:standard:QualifiedDataType:100" xmlns:ram="urn:un:unece:uncefact:data:standard:ReusableAggregateBusinessInformationEntity:100" xmlns:rsm="urn:un:unece:uncefact:data:standard:CrossIndustryInvoice:100" xmlns:udt="urn:un:unece:uncefact:data:standard:UnqualifiedDataType:100" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
     <rsm:ExchangedDocumentContext>
         <ram:BusinessProcessSpecifiedDocumentContextParameter>
-            <ram:ID>A1</ram:ID>
+            <ram:ID/>
```

Comment:
Will lxml expand the tag again? I would have expected to see `<ram:ID></ram:ID>` for an empty tag.

Reply:
Yes, the tags are expanded when any data is added. It's good practice to have empty tags as `<tag/>` rather than `<tag></tag>`.

Reply:
If they are expanded it's fine.